repository/model/project.go

```diff
@@ -0,0 +1,93 @@
+package model
+
+import (
+	"time"
+
+	svcmodel "release-manager/service/model"
+
+	"github.com/google/uuid"
+	"github.com/relvacode/iso8601"
+)
+
+type Project struct {
+	ID                 uuid.UUID          `json:"id"`
+	Name               string             `json:"name"`
+	SlackChannelID     string             `json:"slack_channel_id"`
+	ReleaseAlertConfig ReleaseAlertConfig `json:"release_alert_config"`
+	CreatedAt          iso8601.Time       `json:"created_at"`
```

Comment:
Why not `time.Time`?

Reply:
Thanks! Initially, I had problems with marshaling/unmarshaling into a time.Time variable. But that was probably due to some beginner's mistakes. It works fine with `time.Time` for `timestamp with timezone` column 👍